core/src/main/java/nu/yona/server/analysis/service/GoalConflictMessageDto.java

```diff
@@ -46,10 +47,11 @@
 	private final LocalDateTime activityEndTime;
 	private final UUID goalId;
 	private final UUID activityCategoryId;
+	private final LocalDate startDate; // In the user's time zone
```

Comment:
Suggestion: rename to `activityStartDate` to have it a bit more speaking

Reply:
Agreed.